cfgov/unprocessed/apps/filing-instruction-guide/js/fig-sidenav.js

```diff
@@ -22,6 +22,14 @@ let lastTargetIndex;
 
 document.querySelector('.o-footer').classList.add( 'report-global-footer' );
 
+function scrunchIfNeeded() {
+  let screenX = window.innerWidth || document.documentElement.clientWidth || doc.getElementsByTagName('body')[0].clientWidth;
```

Comment:
Should `doc.` be `document.`? I don't see where `doc` is defined.

Reply:
yes! should be `document`.  `doc` was from something i deleted.

Reply:
If you're just grabbing the body, you can do `document.body` https://developer.mozilla.org/en-US/docs/Web/API/Document/body

Reply:
ah, ok. corrected it.  thanks @anselmbradford